### ISSUE TITLE
Certificates Done Right™ (get rid of java keystore)

### ISSUE THREAD

**Issue by RandomReaper** [cloudio-project/cloudio#2]
Use the **industry standard** .pem files.

**Comment by cm0x4D** [comment]
Yeah, there is a reason why everyone uses JKS files as keystore: Java's SSL implementation only accepts JKS (with mandatory password) as thrust store.

We could get rid of that by using BouncyCastle on the client's side, but I would prefer to keep the dependencies on the client side as small as possible.

PKCS12 is widely used to and has the advantage to keep the certificate and private key together and allows to protect them by a password. So I do not see any advantage using PEM certificates instead.
